### PR TITLE
update MC proposal - discuss remote cluster secret requirements

### PR DESCRIPTION
**Pull request by jmazzitelli**

### CODE REVIEW

File: design/KEPS/multicluster/proposal.md [hunk 1]
```diff
@@ -136,13 +136,18 @@ This process will need to be repeated for every new cluster added. Adding or rem
...
...ioctl command creates a remote cluster secret, it puts a label on the secret to identify the secret as a remote cluster secret. That label name is hardcoded (it is not customizable) with a value of `istio/multiCluster=true`. This is the label that the Kiali Operator/Helm Charts will use to autodiscover remote cluster secrets (e.g. `kubectl get secrets -n istio-system -l istio/multiCluster=true`). This will be a problem if the user has multiple secrets where some were created using the "istio-reader-service-account" SA and some with a Kiali-specific SA. For example, assume the user created a remote cluster secret using istioctl with a custom service account that contains the necessary Kiali permissions (so Kiali can communicate with the remote cluster). Also assume the user followed the Istio docs and created a remote cluster secret using the istioctl defaults so Istio can communicate with the remote cluster. There are now two remote cluster secrets, each labeled with `istio/multiCluster=true`. Kiali will auto-discover and use both. But only one will work with Kiali; the other will cause Kiali to encounter multiple failures and will likely cause Kiali to fail to initialize properly.
```
Comment (leandroberetta) [inline design/KEPS/multicluster/proposal.md:148]: I guess it's very typical that Kiali will be deployed alongside Istiod, and I think users will use istioctl cli to create the remote cluster secrets and then, they will create the kiali's specific remote cluster secrets, so I think we might need to change the label that Kiali use to pick the secrets up?.

**Review by leandroberetta — COMMENTED** [review]
Looks good but I added a comment just to clarify a doubt related to Istio and Kiali secrets.

File: design/KEPS/multicluster/proposal.md (hunk 1 above)
Comment (jmazzitelli) [reply design/KEPS/multicluster/proposal.md:148]: I do mention this on the next line below.. see line 150:

`The idea is that the Kiali tool will generate its own secret with its own identification label (`kiali.io/multiCluster=true`)`

Comment (leandroberetta) [reply design/KEPS/multicluster/proposal.md:148]: Oh, my bad to skip the first part and just read "multiCluster"... sorry.

**Review by leandroberetta — APPROVED** [review]
Looks good.

**Review by jshaughn — APPROVED** [review]
good

**Pull request merged**